tests/resources/image/README.md

```diff
@@ -0,0 +1,28 @@
+# Control Interface Tester workload
+
+The Control Interface Tester workload is used in system test to verify the correct execution of the Ankaios Control Interface.
+The workload reads a commands file, executes the commands and writes an output file containing the result of the commands.
+
+In case a Control Interface instance was not provided to the workload, a `NoAPI` result is written. If the Control Interface was preliminary closed by Ankaios, e.g., due to a protocol error, a `ConnectionClosed` result is provided.
```

Comment:
```suggestion
If no Control Interface instance was provided to the workload, a `NoAPI` result is written. If the Control Interface was preliminary closed by Ankaios, e.g., due to a protocol error, a `ConnectionClosed` result is provided.
```

Reply:
We are at the matter of style here, but when you insist 😉

Reply:
I forgot to mention it as optional. I did not know the "In case a", I thought it is always "In case of ...". And then I checked this with write improvement tool and it suggested just "If no...".